platform/core/src/classes/MetadataProvider.ts

```diff
@@ -22,6 +22,12 @@ class MetadataProvider {
       writable: false,
       value: new Map(),
     });
+    Object.defineProperty(this, 'imageUIDsByImageId', {
```

Comment:
This seems equivalent to 
private readonly imageUIDSByImageId = new Map<string,...>()
This code was originally javascript based, but now that it is typescript, using the behaviour in typescript makes sense.

Reply:
Updated!